packages/sui-test/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@s-ui/test",
+  "version": "1.0.0-beta.10",
+  "description": "",
+  "bin": {
+    "sui-test": "./bin/sui-test.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@s-ui/helpers": "1.5.0",
```

Comment:
We might want to use the major here instead the fixed version?

Reply:
👍